File: app/src/main/java/com/akexorcist/lovelyrecyclerview/adapter/holder/UserDetailViewHolder.java
package com.akexorcist.lovelyrecyclerview.adapter.holder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.akexorcist.lovelyrecyclerview.R;


/**
 * Created by Akexorcist on 10/28/2016 AD.
 */

public class UserDetailViewHolder extends RecyclerView.ViewHolder {
    public TextView tvUserName;

    public UserDetailViewHolder(View itemView) {
        super(itemView);
        tvUserName = (TextView) itemView.findViewById(R.id.tv_user_name);
    }
}

File: app/src/main/java/com/akexorcist/lovelyrecyclerview/adapter/holder/UserDetailViewHolder.kt
package com.akexorcist.lovelyrecyclerview.adapter.holder

import androidx.recyclerview.widget.RecyclerView
import com.akexorcist.lovelyrecyclerview.adapter.model.OrderDetailItem
import com.akexorcist.lovelyrecyclerview.databinding.ViewUserDetailBinding

class UserDetailViewHolder(private val binding: ViewUserDetailBinding) : RecyclerView.ViewHolder(binding.root) {
    fun bind(item: OrderDetailItem.UserDetail) {
        binding.tvUserName.text = item.name
    }
}
